src/Compilers/CSharp/Portable/CSharpCompilationOptions.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
What changed here?
If unicode encoding changed, please revert. Apparently that causes pain during subsequent conflict resolutions.